Task 1/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Task 1</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+      <div class="mainblock">
+        <div class="input">
```

Comment:
This isn't definitely an `.input`, because it is a block. Better usage is `input-wrapper` or `form-field`
```suggestion
        <div class="form-field">
```